src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -83,32 +125,33 @@ You can combine selectors using boolean logic.
 
 ### Value Types
 
-Select subsections by JSON-type or semantic meaning using the following:
-
-* `$string`
-* `$number`
-* `$boolean`
-* `$datetime`
-* `$array`
-* `$object`
-* `$event`
-* `$exception`
-* `$stacktrace`
-* `$frame`
-* `$request`
-* `$user`
-* `$logentry` (also applies to `event.message`)
-* `$thread`
-* `$breadcrumb`
-* `$span`
-* `$sdk`
-
-Examples:
+Select subsections by JSON-type using the following:
+
+* `$string` matches any string value
+* `$number` matches any integer or float value
+* `$datetime` matches any field in the event that represents a timestamp
+* `$array` matches any JSON array value
+* `$object` matches any JSON object
+
+Select known parts of the schema using the following:
+
+* `$exception` matches a single exception instance in `{"exception": {"values": [...]}}`
+* `$stacktrace` matches a stack trace instance
+* `$frame` matches a frame
+* `$request` matches the HTTP request context of an event
+* `$user` matches the user context of an event
+* `$logentry` (also applies to the `message` attribute)
+* `$thread` matches a single thread instance in `{"threads": {"values": [...]}}`
+* `$breadcrumb` matches a single breadcrumb in `{"breadcrumbs": [...]}`
+* `$span` matches a [trace span]({% link _documentation/performance/performance-glossary.md %}#span)
```

Comment:
It's a pity that Jekyll doesn't support checking the reference within the target page, we indeed have to construct the link to `#span` the way it is here, and accept it may become broken without breaking the build (I admit, I don't even know if a bad `{% link %}` would break the build either...).

https://jekyllrb.com/docs/liquid/tags/#link

Reply:
check out `bin/link-check`, unfortunately it is extremely slow as it goes over HTML output and is still Ruby

Reply:
Interesting, thanks for sharing!